scss/navbar.scss

```diff
@@ -0,0 +1,26 @@
+$nav-height: 90px;
+$nav-logo-height: 30px;
+
+.nav-bar {
```

Comment:
you need justify content space between to separate the image from the text and add a padding to the sides so they are not so close to the borders of your screen